.github/workflows/build-pr.yml

```diff
@@ -28,6 +28,27 @@ jobs:
       - name: Download eth2network binaries
         run: go test ./... -v -count=1 -run TestEnsureBinariesAreAvail
 
+      # Close specified ports using lsof before testing / local port list compiled from ./integration/constants.go
```

Comment:
this is from @anixon604's branch. 
It doesn't seem to do anything. Leaving it in , and it will be overridden when he merges